### PR TITLE
Regen gene

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

# Spriting

Adds a gene that makes the user slowly regenerate.
More genes = more better since you need to actually pick instead of taking the 5 meta genes.
# Wiki Documentation

Adds a gene that makes the user slowly regenerate.
# Changelog

:cl:  
rscadd: Adds a gene that makes the user slowly regenerate.
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
podperson healing for only 15 instability sounds terrible balance wise. Make it only heal if you have 10 or less damage or something, so it is only usable for chip damage and has a neat niche, or make it cost way more instability.

**Comment by Simplehorror** [comment]
> podperson healing for only 15 instability sounds terrible balance wise. Make it only heal if you have 10 or less damage or something, so it is only usable for chip damage and has a neat niche, or make it cost way more instability.

sir have you heard of our lord and savior nanites

**Comment by Marmio64** [comment]
> > podperson healing for only 15 instability sounds terrible balance wise. Make it only heal if you have 10 or less damage or something, so it is only usable for chip damage and has a neat niche, or make it cost way more instability.
> 
> sir have you heard of our lord and savior nanites

Little known fact is that EMPs actually counter nanites, has some risk of sabotage, and has a Regen cooldown period if you're using them too often. If you give this a resistance chromosome there's nothing to do about it but put them in the DNA modifier, and it's cost is really cheap for the value.

**Comment by Simplehorror** [comment]
> > > podperson healing for only 15 instability sounds terrible balance wise. Make it only heal if you have 10 or less damage or something, so it is only usable for chip damage and has a neat niche, or make it cost way more instability.
> > 
> > 
> > sir have you heard of our lord and savior nanites
> 
> Little known fact is that EMPs actually counter nanites, has some risk of sabotage, and has a Regen cooldown period if you're using them too often. If you give this a resistance chromosome there's nothing to do about it but put them in the DNA modifier, and it's cost is really cheap for the value.

never seen anyone specifically counter nanites with an EMP, also cloud controller just fixes it anyway

**Comment by CMOisLing** [comment]
I agree nanite passive healing is basically free but that doesnt mean we should add more of it. How about have it be a unique conditional regen. Or how about you take 20% more damage overall but you have a passive 2 brute/burn heal, 1 toxin heal

**Comment by Simplehorror** [comment]
> I agree nanite passive healing is basically free but that doesnt mean we should add more of it. How about have it be a unique conditional regen. Or how about you take 20% more damage overall but you have a passive 2 brute/burn heal, 1 toxin heal

i like that, let me do it

**Comment by ilikesaying** [comment]
>sir have you heard of our lord and savior nanites

Accelerated Regeneration deals stamina damage when repairing.
Bio-Reconstruction consumes nanites at a rapid rate when on, even if there is no damage to repair.
Nanites can be countered by some guy walking into the nanite lab and deleting all the programs/replacing them with weaponized or suppression programs or just destroying the nanite server.

**Comment by boodaliboo** [comment]
MAKE THIS A POD PERSON ONLY MUTATION LIKE HOW FIRE BREATH IS FOUND

**Comment by Simplehorror** [comment]
do humans have any race specific mutations ever since hulk got removed

File: code/datums/mutations/regeneration.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/datum/mutation/human/regeneration
+	name = "Regeneration"
+	desc = "Causes the subject to regenerate passively."
+	instability = 15
```
Comment (Moltijoe) [inline code/datums/mutations/regeneration.dm]: 15 instability is too little for constant healing like this

File: code/datums/mutations/regeneration.dm [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	text_gain_indication = span_notice("You feel better.")
+
+/datum/mutation/human/regeneration/on_life()
+	owner.take_overall_damage(-1, -1)
```
Comment (Moltijoe) [inline code/datums/mutations/regeneration.dm]: should use heal_overall_damage()

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Comment by Simplehorror** [comment]
@Moltijoe done

**Review by Moltijoe — APPROVED** [review]
More in line with balance, probably still fucked, gonna wait on someone else to actually merge it

**Review by Moltijoe — CHANGES_REQUESTED** [review]
after reviewing other mutations, this is still too powerful
cerebral neuroplasticity is 70 instability, this mutation is probably more powerful than that
Either it needs to heal for less or have even higher instability

you could possibly let it be powered if you want to make it possible to have the same strength

Also, the name, description, and gain indicator are all lacking flavour
also it's lacking a text_lose_indication

**Comment by Simplehorror** [comment]
cerebral neuroplasticity is dogshit and I would not use it for 30 instability, but I will work on it

**Comment by JamieD1** [comment]
*opens pr to nerf healing per second*
*opens pr to add regen*

**Comment by CMOisLing** [comment]
> > I agree nanite passive healing is basically free but that doesnt mean we should add more of it. How about have it be a unique conditional regen. Or how about you take 20% more damage overall but you have a passive 2 brute/burn heal, 1 toxin heal
> 
> i like that, let me do it

Damn Guess he never did it

**Comment by Simplehorror** [comment]
why would I do something when the PR is going to be closed

**Pull request closed without merging**